pkg/cfn/builder/iam_test.go

```diff
@@ -166,6 +167,7 @@ var _ = Describe("template builder for IAM", func() {
 		]`))
 
 		Expect(t).To(HaveResourceWithPropertyValue("Role1", "AssumeRolePolicyDocument", expectedAssumeRolePolicyDocument))
+		Expect(t).To(HaveResourceWithProperties("Policy1", "PolicyName"))
```

Comment:
Could you also check the value also, I believe it should be deterministic.

Reply:
It is deterministic. I thought about using it first but it leaks the CloudFormation abstraction. Anyway, I just saw another assertion using an intrinsic function, so I'll add it for this one as well.